grpc-chatapp/client/Makefile

```diff
@@ -0,0 +1,60 @@
+# include .env #if there is a .env file uncomment this line
```

Comment:
Can we have a single Makefile at the root level? It makes it easier to follow and you could add more commands for running the `server` and `client`.